### ISSUE TITLE
WebSSH 13.16.3 TestFlight : Unable to launch a connection, always crash

### ISSUE THREAD

**Issue by isontheline** [isontheline/pro.webssh.net#8]

**Comment by isontheline** [comment]
```
Last Exception Backtrace:
0   CoreFoundation                	0x19de2e654 __exceptionPreprocess + 224 (NSException.m:199)
1   libobjc.A.dylib               	0x19db50bcc objc_exception_throw + 56 (objc-exception.mm:565)
2   UIKitCore                     	0x1a1815618 -[UINavigationController pushViewController:transition:forceImmediate:] + 420 (UINavigationController.m:7309)
3   UIKitCore                     	0x1a1815320 -[UINavigationController pushViewController:animated:] + 656 (UINavigationController.m:7269)
4   WebSSH                        	0x1004c6f68 -[UINavigationController(WYPopover) sizzled_pushViewController:animated:] + 128 (WYPopoverController.m:103)
5   UIKitCore                     	0x1a18ca004 -[_UIViewControllerTransitionCoordinator _applyBlocks:releaseBlocks:] + 264 (UIViewControllerTransitioning.m:1140)
6   UIKitCore                     	0x1a18c67a8 -[_UIViewControllerTransitionContext _runAlongsideCompletions] + 132 (UIViewControllerTransitioning.m:376)
7   UIKitCore                     	0x1a18c64f8 -[_UIViewControllerTransitionContext completeTransition:] + 116 (UIViewControllerTransitioning.m:288)
8   UIKitCore                     	0x1a18d6fd0 __53-[_UINavigationParallaxTransition animateTransition:]_block_invoke.158 + 772 (_UINavigationParallaxTransition.m:502)
9   UIKitCore                     	0x1a238a2e4 -[UIViewAnimationBlockDelegate _didEndBlockAnimation:finished:context:] + 588 (UIView.m:14135)
10  UIKitCore                     	0x1a235e6e8 -[UIViewAnimationState sendDelegateAnimationDidStop:finished:] + 244 (UIView.m:0)
11  UIKitCore                     	0x1a235ebf0 -[UIViewAnimationState animationDidStop:finished:] + 240 (UIView.m:2222)
12  UIKitCore                     	0x1a235ed40 -[UIViewAnimationState animationDidStop:finished:] + 576 (UIView.m:2241)
13  QuartzCore                    	0x1a4944d7c CA::Layer::run_animation_callbacks(void*) + 272 (CALayer.mm:6843)
14  libdispatch.dylib             	0x19daf4524 _dispatch_client_callout + 16 (object.m:495)
15  libdispatch.dylib             	0x19daa65b4 _dispatch_main_queue_callback_4CF$VARIANT$mp + 904 (inline_internal.h:2484)
16  CoreFoundation                	0x19ddac6bc __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 12 (CFRunLoop.c:1749)
17  CoreFoundation                	0x19dda7590 __CFRunLoopRun + 1724 (CFRunLoop.c:3069)
18  CoreFoundation                	0x19dda6ba8 CFRunLoopRunSpecific + 424 (CFRunLoop.c:3192)
19  GraphicsServices              	0x1a7f16344 GSEventRunModal + 160 (GSEvent.c:2246)
20  UIKitCore                     	0x1a1ee23e4 UIApplicationMain + 1932 (UIApplication.m:4823)
21  WebSSH                        	0x1003a4550 main + 88 (main.m:16)
22  libdyld.dylib                 	0x19dc2e8f0 start + 4
```

**Comment by isontheline** [comment]
Closed by 13.16.5
